plugins/woocommerce/src/main/kotlin/org/wordpress/android/fluxc/model/taxes/TaxRateEntity.kt

```diff
@@ -0,0 +1,21 @@
+package org.wordpress.android.fluxc.model.taxes
+
+import androidx.room.Entity
+import org.wordpress.android.fluxc.model.LocalOrRemoteId.LocalId
+import org.wordpress.android.fluxc.model.LocalOrRemoteId.RemoteId
+
+@Entity(
+    tableName = "TaxRate",
+    primaryKeys = ["id", "localSiteId"],
+    )
```

Comment:
```suggestion
@Entity(
    tableName = "TaxRate",
    primaryKeys = ["id", "localSiteId"],
)
```
Just some formatting glitch

Reply:
also fixed https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2852/commits/3cdacc3e21a7fe478de5ec10716630b43e3f9c13